File: Programmers/LV1/정리 필요/정수 제곱근 판별.py
import math

def solution(n):
    answer = 0
    root_n = int(math.sqrt(n))

    if root_n * root_n == n:
        answer = (root_n+1)*(root_n+1)
    else:
        answer = -1

    return answer

n = 121

print((solution(n)))

File: Programmers/LV1/JAVA/정수제곱근판별.java
package Programmers.level1;

public class 정수제곱근판별 {

	public static void main(String[] args) {

		long answer = 0;
		int n = 120;

		Double sqrt = Math.sqrt(n);
		
		if(sqrt == sqrt.intValue()) {
			answer = (long) Math.pow(sqrt+1, 2);
		}else {
			answer = -1;
		}

		
		
		System.out.println(answer);

	}

}
